core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -103,12 +105,13 @@ public class ActiveSyncManager implements Journaled {
   // a local executor service used to launch polling threads
   private final ThreadPoolExecutor mExecutorService;
   private boolean mStarted;
-  private final RetryPolicy mRetryPolicy = ExponentialTimeBoundedRetry.builder()
-        .withMaxDuration(Duration
-            .ofMillis(Configuration.getMs(PropertyKey.MASTER_UFS_ACTIVE_SYNC_RETRY_TIMEOUT)))
-        .withInitialSleep(Duration.ofMillis(100))
-        .withMaxSleep(Duration.ofSeconds(60))
-        .build();
+  private final Supplier<RetryPolicy> mRetryPolicy = () ->
+      ExponentialTimeBoundedRetry.builder()
+          .withMaxDuration(Duration
+              .ofMillis(RETRY_TIMEOUT))
```

Comment:
Suggested to keep get value from configuration to support update this retry timeout value dynamically, we needn't worried about the performance of get value from configuration since the configuration object can maintain the kv in heap memory.

```java
Configuration.getMs(
      PropertyKey.MASTER_UFS_ACTIVE_SYNC_RETRY_TIMEOUT)
```